backend/whisper_with_diarization_as_methods.py

```diff
@@ -348,17 +348,20 @@ def main(process_selected: str, input_file: str, to_english_selection: bool, mod
     # Step 1: Defining input audio path + defining CSV Headers
     input_audio_path = input_file # Insert audio file name and extension here (extensions can include: .mp3, .wav)
     output_csv_headers = [] # Insert your headers here by replacing values of empty strings. Eg: ["Timestamps", "Speaker No", "Text[Eng]"]
+    output_format = ""
     if process_selected == "Transcription Only":
         output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]"]
+        output_format = "transcription"
     else:
         output_csv_headers = ["Timestamps", "Speaker No", "Text[Eng]"]
-    #elif process_selected == "Translation Only":
-    #    output_csv_headers = ["Timestamps", "Speaker No", "Text[Eng]"]
-    #else:
-    #    output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]", "Text[Eng]"]
-    the_date_time = str(datetime.datetime.now()).replace("-", "_").replace(" ", "_").replace(":", "_").replace(".", "_")
-    output_csv_path = destination_selection + "/" + the_date_time + "_" + "streetwhisperapp.csv"
-    translate_to_english = to_english_selection # True denotes that if audio file is not in english, you want to translate text to english. If False, text would be transcribed based on autodetected language from Whisper
+        output_format = "translation"
+
+    #the_date_time = str(datetime.datetime.now()).replace("-", "_").replace(" ", "_").replace(":", "_").replace(".", "_")
+    the_date_time = str(datetime.now().strftime("%H:%M"))
+    audio_path_last_backslash_index = input_file.rfind("/")
+    audio_name = input_file[audio_path_last_backslash_index + 1:]
```

Comment:
I was testing it out and it seems that the backslash is included in the output CSV file